srcs/utils/minishell_utils.c

```diff
@@ -37,3 +41,53 @@ t_minishell		*init_minishell(char **envp)
 	minishell->exit_code = 0;
 	return (minishell);
 }
+
+/*
+**	Termcaps
+**
+** (Text is extracted from @gleal very precious README on this project
+**	and the termcaps documentation)
+**
+**	Termcaps stands for terminal capabilities.
+**
+**		struct termios {
+**			tcflag_t c_iflag;
+**			tcflag_t c_oflag;
+**			tcflag_t c_cflag;
+**			tcflag_t c_lflag;
+**			cc_t c_cc[NCCS];
+**			speed_t c_ispeed;
+**			speed_t c_ospeed;
+**		};
+**
+**			1.	int tcgetattr(int fd, struct termios *termios_p);
+**		we check if there is a terminfo database
+**		(database with all terminal capabilities) for the TERM environment
+**		variable in our envp array and load the functions internally, so that
+**		the tgetstr function works.
+**
+**			2.	int tcsetattr(int fd, int optional_actions,
+**					   			 const struct termios *termios_p);
+**		to check and save in a struct and tputs to activate the 
+**		following capabilities:
+**
+**		ECHOCTL
+**            (not in POSIX) If ECHO is also set, terminal special
+**             characters other than TAB, NL, START, and STOP are echoed
+**             as ^X, where X is the character with ASCII code 0x40
+**             greater than the special character.  For example,
+**             character 0x08 (BS) is echoed as ^H.
+** 			~ECHOCTL : Turns off the standard echo of "CTRL + KEY"
+**		TCSANOW
+**             the change occurs immediately.
+**
+*/

```

Comment:
Interesting. Indeed seems simple after reading the explanation. It was probably not very simple for them though. I think I need to thank Gonçalo next time I see him

Reply:
no no. The documentation is a fucking nightmare. Finding this information is like finding a needle in the sea haha.